src/olympia/devhub/forms.py

```diff
@@ -1448,31 +1417,3 @@ def clean(self):
         if not checker.is_submission_allowed(check_dev_agreement=False):
             raise forms.ValidationError(checker.get_error_message())
         return self.cleaned_data
-
-
-class SingleCategoryForm(forms.Form):
```

Comment:
Possibly intentional, but pointing out that we're dropping the special class for themes which currently only allows a single category (it's a radio select) for one that allows two categories (multiple select).

Reply:
We no longer need `SingleCategoryForm` because `CategoryForm` has the logic to only allow a single category for themes, and we no longer need `CategoryFormSet` because a single `CategoryForm` allows developers to select all the categories for their add-on, since we don't have per-application categories anymore.